src/filename_inspector.py

```diff
@@ -1,88 +1,89 @@
 import fnmatch
 import glob
+import logging
 import os
 import json
 import csv
 
+# Configure logging
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+
 
 def get_input(name: str) -> str:
     value = os.getenv(f'INPUT_{name.upper()}')
     return value
 
 
-def set_output(name: str, value: str):
-    output_file = os.getenv('GITHUB_OUTPUT', None)
-    if output_file:
-        with open(output_file, 'a') as f:
-            f.write(f'{name}={value}\n')
-    else:
-        raise EnvironmentError('GITHUB_OUTPUT environment variable is not set.')
+def set_output(name: str, value: str, default_output_path: str = "default_output.txt"):
```

Comment:
Where did the raise exception move?

Reply:
A new default value will be returned when the variable is not found in the env vars.